src/content/pages/tickets.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Sponsor
+subtitle:
+  Sponsoring EuroPython guarantees you highly targeted visibility
+  and the opportunity to present yourself and your company to one
+  of the largest and most diverse Python communities in Europe and beyond.
+---
+
+import TicketTiers from "../../components/ticket-tiers/ticket-tiers.astro";
+
+
+# EuroPython 2025 Tickets
+<div class="text-justify">
+Discover the latest Python trends, learn from 200+ expert speakers, network with fellow developers, enjoy social events, and explore open spaces to spark new ideas! Organized by the community for the community.
```

Comment:
Only checking about the number here (e.g. if 200+ is accurate).

Reply:
~170 last year, ~192 the year before.

* https://ep2024.europython.eu/speakers/
* https://ep2023.europython.eu/speakers/

Reply:
@hugovk thanks for checking the web! Should we use some number in between, maybe `180+`? 🤔 We don't know the exact number of speakers for this year, but we have roughly the same number of slots as in the previous two years. Some events, such as summits or panel discussions, can have a varying number of speakers.

Reply:
ΙΜΟ is safe to say 180+ based in previous years :). @hugovk thanks for sharing the info!

Reply:
Thanks! Updated in https://github.com/EuroPython/website/pull/1045/commits/2a306b63e71f2d9f5636047c6b93f69ca354bd70